proto/decentraland/social/friendships/friendships.proto

```diff
@@ -0,0 +1,97 @@
+syntax = "proto3";
+package decentraland.social.friendships;
+
+// Validation and anti-spam errors
+enum FriendshipErrorCode {
+  TOO_MANY_REQUESTS_SENT = 0;
+  NOT_ENOUGH_TIME_PASSED = 1;
+  BLOCKED_USER = 2;
+  NON_EXISTING_USER = 3;
+  INVALID_REQUEST = 4;
+  UNKNOWN = 5;
+}
+
+enum EventType {
+  REQUEST = 0;
+  ACCEPT = 1;
+  REJECT = 2;
+  CANCEL = 3;
+  DELETE = 4;
+}
+
+// Just a generic empty type
+message Empty {}
```

Comment:
Let's use the standard `google.protobuf.Empty` instead. [See here for reference and other common types](https://protobuf.dev/reference/protobuf/google.protobuf/#empty).

Reply:
Ooh I didn't know it was the standard, we just thought it'd be better to have our own Empty type hah. Changed it!